santiago_suarez/week_01/mta/js/mta.js

```diff
@@ -0,0 +1,149 @@
+// planTrip('N', 'Times Square', '6', '33rd');
+// The N line has the following stops: Times Square, 34th, 28th, 23rd, Union Square, and 8th
+// The L line has the following stops: 8th, 6th, Union Square, 3rd, and 1st
+// The 6 line has the following stops: Grand Central, 33rd, 28th, 23rd, Union Square, and Astor Place.
+// All 3 subway lines intersect ONLY at Union Square
+// Tell the user the number of stops AND the stops IN ORDER that they will pass
+// through or change at.
+// // console.log() shows output similar to this:
+// // "Your must travel through the following stops on the N line: 34th,
+// 28th, 23rd, Union Square."
+// // "Change at Union Square."
+// // "Your journey continues through the following stops: 23rd, 28th, 33rd."
+// // "7 stops in total."
+window.onload = function() {
+  var stops = [["Times Square", "N"], ["34th", "N"], ["28th", "N"], ["23rd", "N"], ["Union Square", "N"], ["8th", "N"],
+               ["8th", "L"], ["6th", "L"], ["Union Square", "L"], ["3rd", "L"], ["1st", "L"],
+               ["Grand Central", "6"], ["33rd", "6"], ["28th", "6"], ["23rd", "6"], ["Union Square", "6"], ["Astor Place", "6"]
+              ];
+
+  var findIndex = function(station, line) {                                       // Find the index of the station depending on the line
+    var index;
+    for (var i = 0; i < stops.length; i++) {
+      if (stops[i][0] === station && stops[i][1] === line) {
+        index = i;
+      }
+    }
+    return index;
+  };
+
+  var findUSIndex = function(line) {                                              // Find the index of Union Square depending on the line
+    var index;
+    for (var i = 0; i < stops.length; i++) {
+      if (stops[i][1] === line && stops[i][0] === "Union Square") {
+        index = i;
+      }
+    }
+    return index;
```

Comment:
You could move this `return` into the body of the `if` inside the loop, and return as soon as you've found Union Square.